<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LensEffectsWidget</class>
 <widget class="QWidget" name="LensEffectsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>12</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabs_lensEffects">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_gaussianBloom">
      <attribute name="title">
       <string>Bloom</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_16">
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_20">
         <item row="2" column="3" colspan="2">
          <widget class="QPushButton" name="button_gaussianDeleteLayer">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Delete/Disable Bloom image layer</string>
           </property>
           <property name="text">
            <string>Delete layer</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="button_gaussianComputeLayer">
           <property name="toolTip">
            <string>Compute/Update Bloom image layer</string>
           </property>
           <property name="text">
            <string>Compute layer</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="spinBox_gaussianRadius">
           <property name="toolTip">
            <string>Adjust Image length Bloom Radius Value</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QSlider" name="slider_gaussianRadius">
           <property name="toolTip">
            <string>Adjust Image length Bloom Radius</string>
           </property>
           <property name="maximum">
            <number>512</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_gaussianRadius">
           <property name="text">
            <string>Radius</string>
           </property>
           <property name="buddy">
            <cstring>slider_gaussianRadius</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QDoubleSpinBox" name="spinBox_gaussianAmount">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Adjust Bloom amount Value</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QSlider" name="slider_gaussianAmount">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Adjust Bloom amount</string>
           </property>
           <property name="maximum">
            <number>512</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_gaussianAmount">
           <property name="text">
            <string>Amount</string>
           </property>
           <property name="buddy">
            <cstring>slider_gaussianAmount</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_vignetting">
      <attribute name="title">
       <string>Vignetting</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox_vignettingEnabled">
         <property name="toolTip">
          <string>Enable Vignetting</string>
         </property>
         <property name="text">
          <string>Enabled</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_vignettingAmount">
         <property name="text">
          <string>Amount:</string>
         </property>
         <property name="buddy">
          <cstring>slider_vignettingAmount</cstring>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_21">
         <item row="0" column="0">
          <widget class="QLabel" name="label_vignettingMin">
           <property name="text">
            <string>-1.0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_vignettingMed">
           <property name="text">
            <string>0.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_vignettingMax">
           <property name="text">
            <string>+1.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QSlider" name="slider_vignettingAmount">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Adjust Vignetting Amount</string>
           </property>
           <property name="maximum">
            <number>512</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="spinBox_vignettingAmount">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Adjust Vignetting Amount Value</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_chromaticAbberationTab">
      <attribute name="title">
       <string>C. Aberration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBox_caEnabled">
         <property name="toolTip">
          <string>Enable Chromatic Abberation</string>
         </property>
         <property name="text">
          <string>Enabled</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_caAmount">
         <property name="text">
          <string>Amount:</string>
         </property>
         <property name="buddy">
          <cstring>slider_caAmount</cstring>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_22">
         <item row="0" column="0">
          <widget class="QLabel" name="label_caAmountMin">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_caAmountMed">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="checkBox_caAmountMax">
           <property name="text">
            <string>0.1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QSlider" name="slider_caAmount">
           <property name="toolTip">
            <string>Adjust Chromatic Abberation Amount</string>
           </property>
           <property name="maximum">
            <number>512</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="spinBox_caAmount">
           <property name="toolTip">
            <string>Adjust Chromatic Abberation Amount Value</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.500000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_glare">
      <attribute name="title">
       <string>Glare</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QGridLayout" name="gridLayout_23">
         <item row="5" column="1" colspan="6">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="11" column="0" colspan="2">
          <widget class="QPushButton" name="button_glareComputeLayer">
           <property name="toolTip">
            <string>Compute/Update Glare image layer</string>
           </property>
           <property name="text">
            <string>Compute layer</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="checkBox_glareMap">
           <property name="text">
            <string>Use maps</string>
           </property>
          </widget>
         </item>
         <item row="11" column="2" colspan="2">
          <spacer name="hSpacer3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="11" column="4" colspan="3">
          <widget class="QPushButton" name="button_glareDeleteLayer">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Delete/Disable Glare image layer</string>
           </property>
           <property name="text">
            <string>Delete layer</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_lashesMap">
           <property name="text">
            <string>Eye lashes map:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_pupilMap">
           <property name="text">
            <string>Pupil map:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="6">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_pupilMap">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_browsePupilMap">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="1" colspan="6">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="6">
          <widget class="QDoubleSpinBox" name="spinBox_glareRadius">
           <property name="toolTip">
            <string>Adjust Image length Glare Value</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>0.200000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_glareAmount">
           <property name="text">
            <string>Amount:</string>
           </property>
           <property name="buddy">
            <cstring>slider_glareAmount</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="5">
          <widget class="QSlider" name="slider_glareAmount">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Adjust Glare amount</string>
           </property>
           <property name="maximum">
            <number>512</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QDoubleSpinBox" name="spinBox_glareAmount">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Adjust Glare amount Value</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>0.300000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_glareThreshold">
           <property name="text">
            <string>Threshold:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="5">
          <widget class="QSlider" name="slider_glareThreshold">
           <property name="toolTip">
            <string>Just pixels with a brightness above this threshold will be used for the glare effect</string>
           </property>
           <property name="maximum">
            <number>512</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QDoubleSpinBox" name="spinBox_glareThreshold">
           <property name="toolTip">
            <string>Just pixels with a brightness above this threshold will be used for the glare effect</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_glareBlades">
           <property name="text">
            <string>Blades:</string>
           </property>
           <property name="buddy">
            <cstring>spinBox_glareBlades</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLabel" name="glareRadiusLabel">
           <property name="text">
            <string>Radius:</string>
           </property>
           <property name="buddy">
            <cstring>slider_glareRadius</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="3" colspan="4">
          <spacer name="hSpacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>116</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QSpinBox" name="spinBox_glareBlades">
           <property name="toolTip">
            <string>Adjust number of Glare blades used</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="5">
          <widget class="QSlider" name="slider_glareRadius">
           <property name="toolTip">
            <string>Adjust Image length Glare Radius</string>
           </property>
           <property name="maximum">
            <number>512</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="6">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_lashesMap"/>
           </item>
           <item>
            <widget class="QPushButton" name="button_browseLashesMap">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
